README.md

```diff
@@ -7,7 +7,9 @@ This project contains 5 Vulkan layers:
 4. Pipeline cache sideloading layer for supplying pipeline caches to applications that either do not use pipeline caches, or do not initialize them with the intended initial data. The pipeline cache file to load can be specified by setting the `VK_PIPELINE_CACHE_SIDELOAD_FILE` environment variable. The layer creates an implicit pipeline cache object for each device, initialized with the specified file contents, which then gets merged into application pipeline caches (if any), and makes sure that a valid pipeline cache handle is passed to every pipeline creation. This layer does not produce `.csv` log files.
 5. Device memory usage layer. This layer tracks memory explicitly allocated by the application (VkAllocateMemory), usually for images and buffers. For each frame, current allocation and maximum allocation is written to the log file. The output log file location can be set with the `VK_MEMORY_USAGE_LOG` environment variable.
 
-The results are saved as CSV files. Setting the `VK_PERFORMANCE_LAYERS_EVENT_LOG_FILE` environment variable makes all layers append their events (with timestamps) to a single file.
+The results are saved as CSV files. Setting the following environment variables makes all layers append their events (with timestamps) to these file, both contain all the events but in different formats.
```

Comment:
nit: these files

We should further break this down into these two bullets IMO. The new formats are not CSV files like the first sentence suggests. What do you think about starting a new paragraph and describing these two formats in separation?

Reply:
Sounds good!